repository/avnu/metadata.json

```
@@ -0,0 +1,19 @@
+{
+  "id": "AVNU",
```

```suggestion
  "id": "avnu",
```
(needs to be the same as the folder name)